From caec2dbcaa3e58ecb9a01e74a1b8e8a5c51b4dff Mon Sep 17 00:00:00 2001
From: Steven Laver <lavers@google.com>
Date: Tue, 18 Feb 2020 12:56:05 -0800
Subject: [PATCH 185/195] MobileSignalController fix corresponding to upstream
 FeatureConnector change

FeatureConnector was changed upstream, in
platform/frameworks/opt/net/ims:c514e52ea6b0c89ae8f5e2ab8666585ed815900a%5E%21/#F0.
Adjust code in MobileSignalController to match.

Change-Id: Ie21bf189b70e4e16eb9f31e8b85eb6bc0c9f5f20
---
 .../systemui/statusbar/policy/MobileSignalController.java  | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/policy/MobileSignalController.java b/packages/SystemUI/src/com/android/systemui/statusbar/policy/MobileSignalController.java
index 845283fd51b..de06eddaa71 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/policy/MobileSignalController.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/policy/MobileSignalController.java
@@ -133,11 +133,6 @@ public class MobileSignalController extends SignalController<
         int phoneId = mSubscriptionInfo.getSimSlotIndex();
         mFeatureConnector = new FeatureConnector(mContext, phoneId,
                 new FeatureConnector.Listener<ImsManager> () {
-                    @Override
-                    public boolean isSupported() {
-                        return true;
-                    }
-
                     @Override
                     public ImsManager getFeatureManager() {
                         return ImsManager.getInstance(mContext, phoneId);
@@ -155,7 +150,7 @@ public class MobileSignalController extends SignalController<
                         Log.d(mTag, "ImsManager: connection unavailable.");
                         removeListeners();
                     }
-        });
+        }, "?");
 
 
         mObserver = new ContentObserver(new Handler(receiverLooper)) {
-- 
2.30.2

